README.md

```diff
@@ -87,9 +87,11 @@ spark-submit \
 	--class org.hammerlab.guacamole.Guacamole \
 	--verbose \
 	/path/to/target/guacamole-with-dependencies-<x.y.z>.jar \
-	germline-threshold \
-        --reads hdfs:///path/to/reads.bam \
-        --out hdfs:///path/to/result.vcf
+	somatic-joint \
+    		src/test/resources/synth1.normal.100k-200k.withmd.bam \
```

Comment:
this might be somewhat misleading because these are valid relative paths in the repo but will only resolve if you have files at these paths in your HDFS deployment, right?